code/modules/shuttles/shuttle.dm

```diff
@@ -186,3 +186,501 @@
 	for(var/obj/structure/engine_startup_sound/O in get_area(src))
 		playsound(O.loc, 'sound/effects/engine_startup.ogg', 75, 1)
 */
+
+/*************************************
+shuttle_backend.dm
```

Comment:
was this supposed to be removed already?

Reply:
dont mess with shuttle stuff pls

Reply:
It's only moving a file, kind of necessary to get rid of cm_marines